### ISSUE TITLE
warning: reference to compatibility time(); include <time.h> for correct reference

### ISSUE THREAD

**Issue by krytarowski** [dotnet/coreclr#3152]
```
CMakeFiles/paltest_waitformultipleobjectsex_test6.dir/test6.c.o: In function `main':
/tmp/pkgsrc-tmp/wip/coreclr-git/work/coreclr/src/pal/tests/palsuite/threading/WaitForMultipleObjectsEx/test6/test6.c:588: warning: warning: reference to compatibility time(); include <time.h> for correct reference
```

```
$ uname -a
NetBSD chieftec 7.99.25 NetBSD 7.99.25 (GENERIC) #0: Fri Dec 25 20:51:06 UTC 2015  root@chieftec:/tmp/netbsd-tmp/sys/arch/amd64/compile/GENERIC amd64
```

**Comment by krytarowski** [comment]
```
warning: reference to compatibility time(); include <time.h> for correct reference
```

**Comment by krytarowski** [comment]
```
CMakeFiles/paltest_rand_srand_test1.dir/test1.c.o: In function `main':
/tmp/pkgsrc-tmp/wip/coreclr-git/work/coreclr/src/pal/tests/palsuite/c_runtime/rand_srand/test1/test1.c:42: warning: warning: reference to compatibility time(); include <time.h> for correct referenc
```

**Comment by krytarowski** [comment]
Closed via #3149
